src/jbi/model.py

```diff
@@ -0,0 +1,64 @@
+"""
+Python Module for Pydantic Models and validation
+"""
+import importlib
+from inspect import signature
+from types import ModuleType
+from typing import Any, Dict, Optional
+
+from pydantic import Extra, ValidationError, root_validator, validator
+from pydantic_yaml import YamlModel
+
+
+class ActionConfig(YamlModel, extra=Extra.allow):
```

Comment:
Shouldn't it be just `Action` (since we have `Actions` below) ?

Reply:
That works for me. I think the naming has evolved over the time--and I might rename the config files to action.*.yaml as well